ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -210,16 +227,33 @@ export const itwEidIssuanceMachine = setup({
       invoke: {
         src: "getWalletAttestation",
         input: ({ context }) => ({ integrityKeyTag: context.integrityKeyTag }),
-        onDone: {
-          actions: [
-            assign(({ event }) => ({
-              walletInstanceAttestation: event.output
-            })),
-            { type: "storeWalletInstanceAttestation" }
-          ],
-          target: "IpzsPrivacyAcceptance"
-        },
+        onDone: [
+          {
+            guard: "isReissuing",
+            actions: [
+              assign(({ event }) => ({
+                walletInstanceAttestation: event.output
+              })),
+              { type: "storeWalletInstanceAttestation" }
+            ],
+            target: "UserIdentification"
+          },
+          {
+            actions: [
+              assign(({ event }) => ({
+                walletInstanceAttestation: event.output
+              })),
+              { type: "storeWalletInstanceAttestation" }
+            ],
+            target: "IpzsPrivacyAcceptance"
+          }
+        ],
         onError: [
+          {
+            guard: "isReissuingAndSessionExpired",
```

Comment:
We can use the `and` higher guard: `and(["isReissuing", "isSessionExpired"])`.

I have a question on this error case: is it correct to close the issuance (action that pops the navigation stack) and return to idle while the user re-authenticates? Are we sure the user can resume the issuance flow without getting stuck in an "impossible" state? Just to be sure.